bats/tests/snapshots/test-snapshot-list.bats

```diff
@@ -26,15 +26,31 @@ local_setup() {
     assert_output 'No snapshots present.'
 }
 
-@test 'verify snapshot-list output with snapshots' {
+@test 'create three snapshots, allowing for restarts' {
+    # Ensure the app is up and running
+    wait_for_container_engine
+    wait_for_apiserver
+    wait_for_backend
     # Sleep 5 seconds after creating each snapshot so later we can verify
     # that the differences in each snapshot's creation time makes sense.
+
     rdctl snapshot create cows_fish_capers
     sleep 5
+    wait_for_container_engine
+    wait_for_apiserver
+    wait_for_backend
     rdctl snapshot create world-buffalo-federation
     sleep 5
```

Comment:
Are these `sleep` lines necessary?

Reply:
There is a comment on the top that explains it. I was wondering the same thing; that's why I asked for the comment. 😄